test/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ find_package(Git REQUIRED)
 ExternalProject_Add(catch
         PREFIX ${gram_SOURCE_DIR}/test/lib/catch
         GIT_REPOSITORY https://github.com/catchorg/Catch2
-        GIT_TAG master
+        GIT_TAG v2.x
```

Comment:
How does the `.x` work? I don't see this tag in the Catch2 repo and `ExternalProject_Add` documentation doesn't mention any magic that would resolve this to an existing tag.

Reply:
According to CMake documentation, `GIT_TAG` can reference `Git branch name, tag or commit hash`. Thus _v2.x_ references name of stable branch, rather than specific git tag. I will change it to `origin/v2.x` as documentation suggests.

Reply:
Ah, that's right.